pkg/reconciler/reconciler.go

```diff
@@ -796,8 +871,8 @@ func (r *Reconciler) addDefaults(mgr ctrl.Manager, controllerName string) {
 		r.log = ctrl.Log.WithName("controllers").WithName("Helm")
 	}
 	if r.actionClientGetter == nil {
-		actionConfigGetter := helmclient.NewActionConfigGetter(mgr.GetConfig(), mgr.GetRESTMapper(), r.log)
-		r.actionClientGetter = helmclient.NewActionClientGetter(actionConfigGetter)
+		r.actionConfigGetter = helmclient.NewActionConfigGetter(mgr.GetConfig(), mgr.GetRESTMapper(), r.log)
```

Comment:
This should be moved out of the `r.actionClientGetter == nil` if block and into its own, right?

```go
if r.actionConfigGetter == nil {
	r.actionConfigGetter = helmclient.NewActionConfigGetter(mgr.GetConfig(), mgr.GetRESTMapper(), r.log)
}
if r.actionClientGetter == nil {
	r.actionClientGetter = helmclient.NewActionClientGetter(r.actionConfigGetter)
}
```

Reply:
Agree and done